python/pdstools/utils/cdh_utils.py

```diff
@@ -1015,3 +1018,33 @@ def get_latest_pdstools_version():
         return response.json()["info"]["version"]
     except:
         return None
+
+
+def setup_logger():
+    logger = logging.getLogger("PDSTools")
```

Comment:
Would recommend to just use 'pdstools' lowercase, as that's the internal lib name

Reply:
```logger = logging.getLogger()```
Not we are configuring the root level logger in setup_logger because
```pdstools.adm.ADMDatamart.py```
```pdstoosl.app.health_check.Home.py```

The app logger is configured in Home.py but it is not a parent of ADMDatamart in folder hierarchy. So, I am setting the root level logger in the app since it is the common parent.